Formula/openshift-cli.rb

```diff
@@ -23,31 +21,54 @@ class OpenshiftCli < Formula
     sha256 cellar: :any_skip_relocation, x86_64_linux:   "521e52f5ec56cc96bc4ffb721ec55f48e4e73946311d91c813183b0aae88c666"
   end
 
-  depends_on "coreutils" => :build
   depends_on "go" => :build
-  depends_on "socat"
-
   uses_from_macos "krb5"
 
+  on_macos do
+    depends_on "heimdal" => :build
+  end
```

Comment:
Is Apple's Kerberos insufficient?

Why is this only needed at build time? Static linking?

Reply:
Oh, this looks like a difference between using the `make oc` command vs the cross build commands currently used in the formula. After reviewing the makefile, the current way homebrew is doing this looks to be preferred, and as such, I've reverted this back.